modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/headless/batch/engine/resource/v1_0/test/ExportTaskResourceTest.java

```diff
@@ -113,6 +126,99 @@ public void testPostExportTask() throws Exception {
 		}
 	}
 
+	@Test
+	public void testPostExportTaskWithFiltering() throws Exception {
+		try (LogCapture logCapture = LoggerTestUtil.configureLog4JLogger(
+				"com.liferay.batch.engine.internal." +
+					"BatchEngineExportTaskExecutorImpl",
+				LoggerTestUtil.ERROR)) {
+
+			ObjectDefinition objectDefinition =
+				ObjectDefinitionTestUtil.publishObjectDefinition(
+					Collections.singletonList(
+						ObjectFieldUtil.createObjectField(
+							ObjectFieldConstants.BUSINESS_TYPE_TEXT,
+							ObjectFieldConstants.DB_TYPE_STRING,
+							_OBJECT_FIELD_NAME_TEXT)),
+					ObjectDefinitionConstants.SCOPE_COMPANY,
+					TestPropsValues.getUserId());
+
+			String objectDefinitionRESTContextPath =
+				objectDefinition.getRESTContextPath();
+
+			ObjectEntryTestUtil.addObjectEntry(
+				objectDefinition, _OBJECT_FIELD_NAME_TEXT, "TestObject1");
+			ObjectEntryTestUtil.addObjectEntry(
+				objectDefinition, _OBJECT_FIELD_NAME_TEXT, "TestObject2");
+			ObjectEntryTestUtil.addObjectEntry(
+				objectDefinition, _OBJECT_FIELD_NAME_TEXT, "Object3");
+
+			String queryParametersString = StringBundler.concat(
+				"restrictFields=actions&filter=contains%28",
```

Comment:
Please scape the characters by using the method instead of doing that. It will simplify how to read the test

Reply:
https://github.com/liferay/liferay-portal/blob/b6aba40fa9037d15dea3b46d588809a1f9af9275/modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java#L12351

Reply:
Not sure if adding the actions to the restrict fields parameter is relevant to the test

Reply:
I think it is since the actions returned by batch are different from the ones returned by the objects API